scientific_library/tvb/tests/library/datatypes/sensors_test.py

```diff
@@ -69,7 +69,9 @@ def test_sensors(self):
         dummy_surf = SkinAir()
         dummy_surf.vertices = numpy.array(list(range(30))).reshape(10, 3).astype('f')
         dummy_surf.triangles = numpy.array(list(range(9))).reshape(3, 3)
-        dummy_surf.configure()
+
+        with numpy.errstate(all='ignore'):
+            dummy_surf.configure()
```

Comment:
Can't we avoid the errors in `dummy_surf.configure()`, if we define the vertices/triangles in a smarter way ?

Reply:
How? I tried a few different options for the array but line 350 will give the warnings because tri_norm will have 0. values.

Reply:
The fix depends on the actual operation giving error. 
As you say that `tri_norm` are 0, probably they should not be. It should be easy to fill triangle normals with something meaningful (a vector perpendicular on the triangle pointing towards the outside of the object).

Reply:
Avoided the computations generating warnings by defining dummy normals for triangles and vertices. @VinczeRobert please double-check this is reducing the numpy warnings.

Reply:
I checked it again. There are no more numpy warnings when running the library tests.